integration/integration_plugins_test.go

```diff
@@ -0,0 +1,66 @@
+// +build integration
+
+package main
+
+import (
+	delugeclient "github.com/gdm85/go-libdeluge"
+	"testing"
+	"time"
+)
+
+func waitForPluginEnabled(t *testing.T, name string) {
```

Comment:
Is this necessary because enabling plugins is asynchronous?

Reply:
At some point it seemed it was, but I was wrong it works without. This has been removed now.